docs/modules/ROOT/pages/admin-api-reference.adoc

```diff
@@ -179,6 +179,54 @@ The same call can be made using `defender-admin-client` passing the contract and
 await client.getProposal(contractId, proposalId);
 ```
 
+[[simulate-endpoint]]
+=== Simulate a Proposal
+
+Proposals can be simulated via the API. The results of a simulation (`SimulationResponse`) will be stored and can be retrieved with the `getProposalSimulation` endpoint.
+
+```js
+const proposal = await client.getProposal(contractId, proposalId);
+
+// import the ABI and create an ethers interface
+const contractInterface = new utils.Interface(contractABI);
```

Comment:
Thoughts on

```suggestion
const contractInterface = new ethers.utils.Interface(contractABI);
```

?

I know it says ethers in the comment directly above but when I first saw this I thought it was some utils associated with the `defender-client` package